### ISSUE TITLE
cleverer sampling in the task

### ISSUE THREAD

**Issue by kmader** [chestrays/jupyanno#37]
![image](https://user-images.githubusercontent.com/116120/44144750-06688dcc-a089-11e8-8df0-b8bbf05f0e05.png)

Right now the task is too biased and it would be better if sklearn.dummy got a worse score

**Comment by betatim** [comment]
I would leave this to a later point when we have time to also trial how we'd use some form of active learning for the sampling.

Things will change again if we have questions where multiple labels are allowed. So maybe finding a good way to have the various configurations of `DummyClassifier` make a submission would be good so that we can see a reasonable baseline no matter what the mixture of images/questions/labels is and without having to think about it explicitly.
